lib/orbf/rules_engine/data/package.rb

```diff
@@ -31,13 +31,15 @@ def self.assert_valid(package_kind)
       attr_reader :activities, :kind, :rules, :frequency, :code,
                   :main_org_unit_group_ext_ids, :target_org_unit_group_ext_ids,
                   :groupset_ext_id, :dataset_ext_ids,
-                  :matching_groupset_ext_ids, :include_main_orgunit, :loop_over_combo
+                  :matching_groupset_ext_ids, :include_main_orgunit, :loop_over_combo,
```

Comment:
Starting to think we could use some docs about the various options of the package.

Reply:
I did some slides in google drive to explain some of them (eg zones and main/target/groupsets ), it would be great to have them as the taskr doc, referencable by the hesabu manager.